src/Support/helpers.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
```

Comment:
Do we need this helper class? I think we can just put them in Support/Util.php class

Reply:
I feel this is a bit more clean (shorter) as we don't need the namespace. There are lots of those logic in the code, I did not replace them all but I think this way would look better.